bcos-framework/bcos-framework/storage/Common.h

```diff
@@ -207,13 +207,108 @@ struct Condition
             return cmpStr + " " + value;
         }
     };
+
+    std::pair<std::optional<cond>, std::optional<Condition>> 
+        simplify(const std::string& defaultStartKey, const std::string& defaultEndKey)
```

Comment:
Maybe simplify only make condition more simple not change the original meaning.
the `defaultStartKey` and `defaultEndKey` can be modify by `GE` and `LE`?